app/models/digital_object/base.rb

```diff
@@ -21,7 +21,7 @@ class DigitalObject::Base
   
   def require_subclass_override!; raise 'This method must be overridden by a subclass'; end
   
-  def initialize()
+  def initialize(*args)
```

Comment:
Do you mind if I just remove all cases where we pass the args hash?  This class and its subclasses aren't supposed to accept any args in the initializer.  I don't want the method signature to imply that they do.